### DISCUSSION TITLE
Disease susceptibility

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#2672] [Mondo design decisions]
Super ticket/discussion for classification of 'disease susceptibility _(in progress)_

Open questions:
1. Should 'disease susceptibility' be obsoleted (https://github.com/monarch-initiative/mondo/issues/496) 
2. How should terms that are a susceptibility to a disease, like hereditary nonpolyposis colon cancer be classified? (See ticket: https://github.com/monarch-initiative/mondo/issues/1509)
3. Should disease susceptibility not be a subclass of the disease? (see: https://github.com/monarch-initiative/mondo/issues/2334)
4. Split OMIM terms that combine disease and disease susceptibility? (see: https://github.com/monarch-initiative/mondo/issues/2562 and https://github.com/monarch-initiative/mondo/issues/2634). 

Related tickets:
- MELANOMA, UVEAL https://github.com/monarch-initiative/mondo/issues/2656
- review susceptibility terms in Mondo, https://github.com/monarch-initiative/mondo/issues/699
- osteoarthritis susceptibility exact matching osteoarthritis, https://github.com/monarch-initiative/mondo/issues/1395
- missing parent: hereditary nonpolyposis colon cancer (MONDO:0018630), https://github.com/monarch-initiative/mondo/issues/1509
-  childhood absence epilepsy, https://github.com/monarch-initiative/mondo/issues/1535
- [Merge] MONDO:0012461 bulimia nervosa, susceptibility to, 2, https://github.com/monarch-initiative/mondo/issues/1705
- Merge status? MONDO:0024462 familial cutaneous melanoma, https://github.com/monarch-initiative/mondo/issues/2326
- disease susceptibility should not be a subclass of the disease, https://github.com/monarch-initiative/mondo/issues/2334

Related docs:
- [susceptibility terms](https://docs.google.com/spreadsheets/d/13Fs1ajbRwaCWNhDgcN0kIfhUmQ2ioCFBj0CwZIEAbZQ/edit#gid=948762561) spreadsheet generated in 2019.

All susceptibility tickets are [here](https://github.com/monarch-initiative/mondo/issues?q=is%3Aissue+label%3Asusceptibility)

**Comment by maglott** [comment]
FWIW, I lean toward making a distinction between susceptibility and the disorder itself.  Perhaps the distinction is clearest for infections, where the infection, with an etiology of an infectious agent, is not the same as the susceptibility to an infection, with a likely etiology of some genetic difference in an immune pathway.

**Comment by nicolevasilevsky** [comment]
Some of the susceptibility terms may be obsoletion candidates. Note to self- this is something I can review and come up with a list of suggested obsoletion candidates.

**Comment by nicolevasilevsky** [comment]
On the Mondo curators call on 05 March 2021, Nicole and Chris implemented the following changes:

- revised some familial cancers (from OMIM or NCIT) to use the [susceptibility pattern](https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/inherited_susceptibility.yaml) (see below). This included:
  - Relabel terms to: '[disease] susceptibility'
  - Add (or revise) logical definition to: 'inherited disease susceptibility' and ('predisposes towards' some %s)

Revised terms:
`MONDO_0024462 'cutaneous melanoma susceptibility'`
`MONDO_0003582 'hereditary breast ovarian cancer syndrome'`
`MONDO_0024462 'cutaneous melanoma susceptibility'`
`MONDO_0011789 'meningioma susceptibility'` 
`MONDO:0016248 ovarian cancer susceptibility`

action item:
DL query: 'neoplasm (disease)' and 'has modifier' some inherited
Look at these children and classify them according to sucept. pattern

Ticket: https://github.com/monarch-initiative/mondo/issues/2784

**Comment by maglott** [comment]
If Mondo changes labels, rather than creating new records, it seems to be changing the meaning of the Mondo record, i.e., from the disease to the susceptibility to the disease. Does this mean that Mondo will not retain a record specific to, for example, 'cutaneous melanoma' as well as 'cutaneous melanoma susceptibility.  The XRefs to NCIT and UMLS will all need to be removed from 'cutaneous melanoma'. MedGen will have to do a bit of cleanup, because we don't want to redefine cutaneous melanoma to cutaneous melanoma susceptibility based on following the Mondo ID and its previous label.

**Reply by nicolevasilevsky** [reply]
This is a really good question, @maglott. Let me talk to @cmungall about this and get back to you. These changes aren't in the released file, so please don't start any clean up in MedGen yet, we'll be sure to coordinate with you before any changes are released. :)

@maglott 
Noted about cleaning up the xrefs in Mondo, thank you!

We will maintain classes like 'cutaneous melanoma' (MONDO_0005012). These are used in the logical axioms in Mondo - 'cutaneous melanoma susceptibility' has the logical axiom:
'inherited disease susceptibility' and ('predisposes towards' some 'cutaneous melanoma (disease)')

@cmungall should we obsolete the familial cancers and create new classes that are '[disease] susceptibility'?
I think it is probably best to obsolete the old class and create a new class. @maglott, if I understand correctly, that would be best for your workflow, right?

**Reply by cmungall** [reply]
yes, we will retain cutaneous melanoma, no change in meaning or label

the case of the familial forms is interesting. By one argument, there is no significant change in concept here, we are just clarifying ontologically. But I think this argument is fairly weak. Obsoletion and new ID is safest

**Comment by maglott** [comment]
Hi Nicole, 
Yes, for our workflow if a change in label alters the definition of the concept represented by the Mondo ID, we would prefer a new Mondo ID for the new label (and likely retain the existing Mondo ID/label if the concept is in scope for Mondo)

**Reply by nicolevasilevsky** [reply]
Sounds good. I will obsolete these terms and create new classes. Thanks for your feedback!
